### ISSUE TITLE
Add package-specific gubbins

### ISSUE THREAD

**Issue by valeriupredoi** [valeriupredoi/PyActiveStorage#20]
- `pyproject.toml`
- CI to build the conda-forge and PyPi packages
- @davidhassell @bnlawrence do we want an executable for our package here or all should be done via imports?

**Comment by bnlawrence** [comment]
It can only ever be done via import surely? This is just a thing that will need to be loaded instead of (as well as) NetCDF4 or zarr?

**Comment by valeriupredoi** [comment]
indeed, that's what I was thinking too, a library indeed; we can add execs in the future tho, we can make this a netCDF processing tool like cdo or nco :grin:

**Comment by bnlawrence** [comment]
I would think we'd want NCO and CDO and friends to implement it when it has wider support, rather than build our own cuckoo to go into others workflows. So, for now, no executables :-)
